src/app/components/pages/Groups.tsx

```diff
@@ -501,7 +538,7 @@ export const Groups = ({ user }: { user: RegisteredUserDTO }) => {
     if (selectedGroup?.id) {
       getGroupMembers(selectedGroup.id);
     }
-  }, [selectedGroup?.id]); // This can't just be group, because group changes when the members change, causing an infinite reload loop
+  }, [selectedGroup?.id, getGroupMembers]); // This can't just be group, because group changes when the members change, causing an infinite reload loop
```

Comment:
is this comment still useful?

Reply:
Yes I feel this is relevant to leave; if I changed the dependency of this useEffect to be [selectedGroup, getGroupMembers] it goes into an infinite reload.  I can remove if you think it unnecessary but I found it useful at least